app/src/main/res/values/strings.xml

```diff
@@ -23,4 +23,5 @@
   <string name="location_tracking_activity">Location Tracking</string>
   <string name="scale_bar_activity">Scale Bar</string>
   <string name="street_view">Street View</string>
+  <string name="custom_location_button">Custom Location Button</string>
```

Comment:
`custom_my_location_button` and "Custom My Location Button" to match the full name of the control ("My Location")